IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/include/user_settings.h

```diff
@@ -584,7 +986,8 @@ Turn on timer debugging (used when CPU cycles not available)
     #define WOLFSSL_BASE16
 #else
     #if defined(USE_CERT_BUFFERS_2048)
-        /* Be sure to include in app when using example certs: */
+    	#define USE_CERT_BUFFERS_256
+    	/* Be sure to include in app when using example certs: */
```

Comment:
fix hard tabs in the indentation here.

Reply:
ah yes, interesting. Visual Studio _usually_ converts to spaces at save time. Good catch. Fixed